src/RazorSdk/Targets/Microsoft.NET.Sdk.Razor.ScopedCss.targets

```diff
@@ -108,7 +108,18 @@ Integration with static web assets:
 
   <ItemGroup>
     <Content Remove="@(ScopedCssInput)" />
-    <Content Include="@(ScopedCssInput)" Pack="false" CopyToPublishDirectory="Never" />
+    <!--
+      See https://github.com/dotnet/maui/issues/791 for details.
+      PublishFolderType="None" prevents content items frrom being bundled as part of MAUI iOS targets.
+      ExcludeFromContentCheck="true" prevents content items from being inspected by MAUI Android targets for warnings.
+    -->
+
+    <Content
+      Include="@(ScopedCssInput)"
+      Pack="false"
+      CopyToPublishDirectory="Never"
+      PublishFolderType="None"
+      ExcludeFromContentCheck="true" />
```

Comment:
I'm a bit uneasy about having this check here. I think the WebContentItem is ok, but I rather have something on the Maui targets than add something Maui specific to the Razor SDK.

Reply:
I can play around with this - what happens today is that .razor.css start off as being `None` but get transformed to `Content` because of this addition. I wanted to avoid  relying on the target order between Razor's and Xamarin's targets since it might change with either of us knowing and break in subtle ways. 

One option might be to update .razor.css during global evaluation if the feature is enabled, but let me try it out to see how it works.

Reply:
> I can play around with this - what happens today is that .razor.css start off as being None but get transformed to Content because of this addition. I wanted to avoid relying on the target order between Razor's and Xamarin's targets since it might change with either of us knowing and break in subtle ways.

If there is a concrete target on the Maui SDK we can hook into, we can have this logic contained within our targets in Maui.

Reply:
Should this be marked with `WebContentItem="true"` so that we can identify it on a Blazor Maui target?